gdcdictionary/schemas/_terms.yaml

```diff
@@ -6105,3 +6106,332 @@ morphologic_architectural_pattern:
+      term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C97229"
+
+exposure_source:
+  common:
+    description: >-
+      The source or location where the patient was exposed.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+hpv_strain:
+  common:
+    description: >-
+      The specific hpv strain being tested.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+lesions_treated_number:
+  common:
+    description: >-
+      The number of lesions treated.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+max_tumor_bulk_site:
+  common:
+    description: >-
+      The site of the tumor where the dimension or diameter is larger than any other
+      part of the tumor.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+number_of_fractions:
+  common:
+    description: >-
+      The total number of divided radiation doses received.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+prescribed_dose:
+  common:
+    description: >-
+      A quantity of an agent prescribed to the study participant.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+protocol_identifier:
+  common:
+    description: >-
+      A sequence of letters, numbers, or other characters that uniquely identifies
+      a study protocol.
+    termDef:
+      term:
+      source: NCIt
+      cde_id: C132299
+      cde_version: 22.05e
+      term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C132299"
+
+treatment_dose_max:
+  common:
+    description: >-
+      A maximum quantity of an agent administered to the study participant.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+tumor_level_prostate:
+  common:
+    description: >-
+      The level(s) of the prostate from which the tumor originated.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+uicc_clinical_m:
+  common:
+    description: >-
+      The UICC TNM Classification is an anatomically based system that records the
+      primary and regional nodal extent of the tumor and the absence or presence of
+      metastases. The clinical M category describes the presence or otherwise of distant
+      metastatic spread as determined during a clinical exam.
+    termDef:
+      term:
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+uicc_clinical_n:
+  common:
+    description: >-
+      The UICC TNM Classification is an anatomically based system that records the
+      primary and regional nodal extent of the tumor and the absence or presence of
+      metastases.  The clinical N category describes the regional lymph node involvement
```

Comment:
2 spaces between the sentences here.

Reply:
I just removed all double spaces in the file. These should look better now.